plugins/modules/get_certificate.py

```diff
@@ -99,6 +99,14 @@
         - The default value V(false) is B(deprecated) and will change to V(true) in community.crypto 3.0.0.
       type: bool
       version_added: 2.12.0
+    tls_ctx_options:
+      description:
+        - TLS CTX options (TLS/SSL OP flags) to use for the request.
```

Comment:
I would write 'context' out at least once:
```suggestion
        - TLS context options (TLS/SSL OP flags) to use for the request.
```

Reply:
Makes sense. Changing both instances in the description. Would you prefer `tls_ctx_options` to be changed to `tls_context_options` as well?

Reply:
I think `ctx` in the option name is fine, as long as the documentation at least once mentions `context` :)